api/info

```diff
@@ -21,19 +21,21 @@
 
 #define LINEWIDTH 80
 
+#include <kprint>
+
 // Informational prints
 #ifndef NO_INFO
-#define INFO(FROM, TEXT, ...) printf("%13s ] " TEXT "\n", "[ " FROM, ##__VA_ARGS__)
-#define INFO2(TEXT, ...) printf("%16s" TEXT "\n"," ", ##__VA_ARGS__)
-#define CENTER(TEXT) printf("%*s%*s\n",LINEWIDTH/2 + (int)strlen(TEXT)/2,TEXT,LINEWIDTH/2-(int)strlen(TEXT)/2,"")
+#define INFO(FROM, TEXT, ...) kprintf("%13s ] " TEXT "\n", "[ " FROM, ##__VA_ARGS__)
+#define INFO2(TEXT, ...) kprintf("%16s" TEXT "\n"," ", ##__VA_ARGS__)
+#define CENTER(TEXT) kprintf("%*s%*s\n",LINEWIDTH/2 + (int)strlen(TEXT)/2,TEXT,LINEWIDTH/2-(int)strlen(TEXT)/2,"")
 #include <string>
-#define FILLINE(CHAR) printf("%s\n",std::string(LINEWIDTH,CHAR).c_str())
-#define CAPTION(TEXT) FILLINE('='); printf("\n"); CENTER(TEXT); printf("\n"); FILLINE('=')
+#define FILLINE(CHAR) kprintf("%s\n",std::string(LINEWIDTH,CHAR).c_str())
```

Comment:
std::string may use malloc and requires heap to be initialised, so you could use`std::pmr::string` instead (which should be set up by that point)

Reply:
> std::string may use malloc and requires heap to be initialised, so you could use`std::pmr::string` instead (which should be set up by that point)

Good point! And funny that this is used exactly at the point when std::string is possible:  

```
<vm> [ Field Medic ] ⛑️  Malloc and brk are functional
<vm> ================================================================================
```
But much better to make it safe, will do.

Reply:
Done!